here-naksha-lib-ext-manager/src/main/java/com/here/naksha/lib/extmanager/ExtensionCache.java

```diff
@@ -134,9 +134,11 @@ private void publishIntoCache(KVPair<Extension, File> result, ExtensionConfig ex
       loaderCache.put(extension.getId(), new KVPair<Extension, ClassLoader>(extension, loader));
       PluginCache.removeExtensionCache(extension.getId());
       logger.info(
-          "Extension {},{} is successfully loaded into the cache.",
+          "Extension [ID,VERSION,JAR,ENV] {},{},{},{} is successfully loaded into the cache.",
```

Comment:
I suggest rephrasing the log as currently it is confusing to read. May be something like:
`Extension id={}, version={} is successfully loaded into the cache, using Jar at {} for env={}.`

Reply:
fixed